src/main/scala/com/netflix/edda/aws/S3CurrentDatastore.scala

```diff
@@ -63,11 +63,9 @@ class S3CurrentDatastore(val name: String) extends Datastore {
   // of sync with the datastore
   val writeDynamo = {
     val client = new AwsClient(account).dynamo
-    val config = new com.amazonaws.ClientConfiguration().withMaxErrorRetry(0)
-    client.setConfiguration(config)
```

Comment:
Edda would not build with the new version of the SDK with that.  Removing it did not seem to have an affect on the application, though I am only using Mongo and not S3.